tests/integration-tests/tests/dcv/test_dcv/test_dcv_configuration/verify_no_core_files.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -x
+
+CRASHDIR=/var/crash
+
+# There are no crash files if the directory doesn't exist
+[ -d $CRASHDIR ] || exit 0
+
+# Exit nonzero if there are any files
+crash_files="`ls -A $CRASHDIR`"
```

Comment:
minor: It's preferable to use `$(..)` in place of backticks, see: http://mywiki.wooledge.org/BashFAQ/082

Reply:
Done.